wikiconv/run_pipeline.sh

```diff
@@ -21,49 +21,114 @@
 
-cd ingest_revisions
-. ${pathToVirtualEnv}/bin/activate
-python dataflow_main.py --setup_file ./setup.py --ingestFrom=cloud --language=${language} --dumpdate=${dumpdate} --cloudBucket=${cloudBucket}/raw-downloads/${language}-${dumpdate} --output=gs://${cloudBucket}/ingested --project ${cloudProject} --bucket ${cloudBucket}|| exit 1
-deactivate
-cd ..
+# Ingest dump into JSON
 
-# Initialize Page States
+if (( PHASE2 )); then
+  cd ingest_revisions
+  . "${pathToVirtualEnv}/bin/activate"
+  python dataflow_main.py --setup_file ./setup.py --ingestFrom=cloud \
+    --language="${language}" --dumpdate="${dumpdate}" \
+    --cloudBucket="${cloudBucket}/raw-downloads/${language}-${dumpdate}" \
+    --output="gs://${cloudBucket}/ingested" --project "${cloudProject}" \
+    --bucket "${cloudBucket}"
+  deactivate
+  cd ..
+fi
 
-gsutil -m rm -r gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/next_stage/*
-gsutil -m rm -r gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/*
-gsutil -m rm -r gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/bakup/*
+if (( PHASE3 )); then
+  # Initialize Page States
+  gsutil -m rm -r \
+    "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/next_stage/*" \
+    "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/*" \
+    "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/bakup/*" || true
 
-gsutil -m cp empty_file gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/last_rev
-gsutil -m cp empty_file gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/page_states
-gsutil -m cp empty_file gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/error_log
+  gsutil -m cp empty_file \
+    "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/last_revisions/last_rev"
+  gsutil -m cp empty_file \
+    "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/page_states/page_states"
+  gsutil -m cp empty_file \
+    "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current/error_logs/error_log"
 
-# Start Reconstruction
-cd conversation_reconstruction
-. ${pathToVirtualEnv}/bin/activate
-python dataflow_main.py --input gs://${cloudBucket}/ingested/${dumpdate}-${language}/*/revisions*.json --setup_file ./setup.py --output_name ${language}${dumpdate} --process_file process_tmp_${language}_${dumpdate} --project ${cloudProject} --bucket ${cloudBucket}|| exit 1
-deactivate
-cd ..
+  # Start Reconstruction
+  cd conversation_reconstruction
+  . ${pathToVirtualEnv}/bin/activate
+  python dataflow_main.py \
+    --setup_file ./setup.py \
+    --input_state "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/current" \
+    --input_revisions "gs://${cloudBucket}/ingested/${dumpdate}-${language}/*/revisions*.json" \
+    --output_state "gs://${cloudBucket}/process_tmp_${language}_${dumpdate}/next_stage" \
+    --output_conversations "gs://${cloudBucket}/conversations-${language}${dumpdate}" \
+    --runner DirectRunner \
```

Comment:
Maybe change this to DataflowRunner?

Reply:
That explains why it wasn't launching any jobs :-)
Done.